profiler/src/test/java/com/splunk/opentelemetry/profiler/ConfigurationTest.java

```diff
@@ -0,0 +1,41 @@
+package com.splunk.opentelemetry.profiler;
+
+import io.opentelemetry.instrumentation.api.config.Config;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+class ConfigurationTest {
```

Comment:
this change seems unrelated to the logging of the config.

Reply:
It is. I had factored it out thinking that I would use it in the logging, but it was misleading in the logs, so I didn't end up using it there...but I felt like it was still a good change so I left it. Do you think it should be removed?

Reply:
doesn't matter to me too much. I was just confused to see this here when it didn't appear to have anything to do with the rest of the PR. :)